docs/user-manual/index.adoc

```diff
@@ -0,0 +1,136 @@
+:idprefix:
+:idseparator: -
+
```

Comment:
Should probably have a note comment in here, indicating people need to update the separate *_book.adoc* file as well for the PDF, mirroring the comment in that file.